package.json

```diff
@@ -74,6 +74,6 @@
     "shortid": "^2.2.8",
     "socket.io": "^2.0.4",
     "store": "^2.0.12",
-    "yup": "^0.25.1"
+    "yup": "^0.27.0"
```

Comment:
if you're interested in updating all packages, use `yarn upgrade`

Reply:
I updates all the packages using `npm-check-updates` npm package.

Reply:
Babel is breaking on updating every package. I am trying to resolve it